source/config.js

```diff
@@ -11,6 +11,10 @@ var defaultConfig = {
 
   // The base URL ungit will be accessible from.
   urlBase: "http://localhost",
+  
+  // The URL root path under which ungit will be accesible.
+  // Either blank or starting with a slash and no trailing slash
```

Comment:
Is this constraint enforced anywhere in the code? Seems like a user could easily violate this constraint without realizing it.